### PR TITLE
fix: routing form delete button not rendered

### LINKED ISSUE #7412: Routing Form: Delete button does not work
Found a bug? Please fill out the sections below. 👍

### Issue Summary

Clicking on `Yes, delete Form` does not delete the form, and the CSS needs fixing too

https://www.loom.com/share/92f22a00b8074b8ca733445a1f06ebdb

### PR DESCRIPTION
**Pull request by G3root**
## What does this PR do?

Fixes #7412 

after:
Screenshot 2023-02-27 15:43:20

**Environment**: Staging(main branch) / Production

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

**Review by hariombalhara — APPROVED** [review]

**Review by emrysal — APPROVED** [review]

**Review by ciaranha — COMMENTED** [review]
Minor copy update suggestions, otherwise looks good!

CleanShot 2023-02-27 at 12 48 25@2x

**Comment by G3root** [comment]
@Jaibles updated the copy .

Screenshot 2023-02-27 19:52:05

File: apps/web/public/static/locales/fr/common.json [hunk 1]
```diff
@@ -1620,7 +1620,7 @@
   "form_deleted": "Formulaire supprimé",
   "delete_form": "Supprimer le formulaire",
   "delete_form_action": "Oui, supprimer le formulaire",
-  "delete_form_confirmation": "Voulez-vous vraiment supprimer ce formulaire ? Toute personne avec qui vous avez partagé le lien ne pourra plus passer de réservation en l'utilisant. De plus, toutes les réponses associées seraient supprimées.",
+  "delete_form_confirmation_1": "Voulez-vous vraiment supprimer ce formulaire ? Toute personne avec qui vous avez partagé le lien ne pourra plus passer de réservation en l'utilisant. De plus, toutes les réponses associées seraient supprimées.",
```
Comment (hariombalhara) [inline apps/web/public/static/locales/fr/common.json:1623]: why this change? Here we are not breaking the string in 2 parts and just assigning the complete string to confirmaton_1

File: apps/web/public/static/locales/ar/common.json [hunk 2]
```diff
@@ -1605,7 +1605,7 @@
   "form_deleted": "تم حذف النموذج",
   "delete_form": "حذف النموذج",
   "delete_form_action": "نعم، احذف النموذج",
-  "delete_form_confirmation": "هل أنت متأكد من أنك تريد حذف هذا النموذج؟ لن يتمكن أي شخص تشارك الرابط معه من الحجز بعد الآن. كما ستحذف جميع الردود المرتبطة بذلك.",
+  "delete_form_confirmation_1": "هل أنت متأكد من أنك تريد حذف هذا النموذج؟ لن يتمكن أي شخص تشارك الرابط معه من الحجز بعد الآن. كما ستحذف جميع الردود المرتبطة بذلك.",
```
Comment (hariombalhara) [inline apps/web/public/static/locales/ar/common.json:1608]:
```suggestion
  "delete_form_confirmation": "هل أنت متأكد من أنك تريد حذف هذا النموذج؟ لن يتمكن أي شخص تشارك الرابط معه من الحجز بعد الآن. كما ستحذف جميع الردود المرتبطة بذلك.",
```

File: apps/web/public/static/locales/da/common.json [hunk 3]
```diff
@@ -1611,7 +1611,7 @@
   "form_deleted": "Formular slettet",
   "delete_form": "Slet Formular",
   "delete_form_action": "Ja, slet formular",
-  "delete_form_confirmation": "Er du sikker på, at du vil slette denne formular? Enhver, som du har delt linket med, vil ikke længere kunne booke via dette. Desuden ville alle tilknyttede svar blive slettet.",
+  "delete_form_confirmation_1": "Er du sikker på, at du vil slette denne formular? Enhver, som du har delt linket med, vil ikke længere kunne booke via dette. Desuden ville alle tilknyttede svar blive slettet.",
```
Comment (hariombalhara) [inline apps/web/public/static/locales/da/common.json:1614]:
```suggestion
  "delete_form_confirmation": "Er du sikker på, at du vil slette denne formular? Enhver, som du har delt linket med, vil ikke længere kunne booke via dette. Desuden ville alle tilknyttede svar blive slettet.",
```

File: apps/web/public/static/locales/de/common.json [hunk 4]
```diff
@@ -1605,7 +1605,7 @@
   "form_deleted": "Formular gelöscht",
   "delete_form": "Formular löschen",
   "delete_form_action": "Ja, Formular löschen",
-  "delete_form_confirmation": "Sind Sie sicher, dass Sie dieses Formular löschen möchten? Jede Person, mit dem Link kann nicht mehr mit dem Formular buchen. Außerdem wurden alle zugehörigen Antworten gelöscht.",
+  "delete_form_confirmation_1": "Sind Sie sicher, dass Sie dieses Formular löschen möchten? Jede Person, mit dem Link kann nicht mehr mit dem Formular buchen. Außerdem wurden alle zugehörigen Antworten gelöscht.",
```
Comment (hariombalhara) [inline apps/web/public/static/locales/de/common.json:1608]:
```suggestion
  "delete_form_confirmation": "Sind Sie sicher, dass Sie dieses Formular löschen möchten? Jede Person, mit dem Link kann nicht mehr mit dem Formular buchen. Außerdem wurden alle zugehörigen Antworten gelöscht.",
```

**Review by emrysal — CHANGES_REQUESTED** [review]
Please clarify the translation issue here; I do not understand the multiple bullet point situation and differences thereof and neither will translators, so this needs addressing before merge.

**Comment by G3root** [comment]
> Please clarify the translation issue here; I do not understand the multiple bullet point situation and differences thereof and neither will translators, so this needs addressing before merge.

Any suggestions for naming the i18n keys here, which are more readable ?

**Review by emrysal — APPROVED** [review]

**Pull request merged**